packages/react-scripts/scripts/start.js

```diff
@@ -85,6 +85,12 @@ choosePort(HOST, DEFAULT_PORT)
       }
       console.log(chalk.cyan('Starting the development server...\n'));
       openBrowser(urls.localUrlForBrowser);
+      if (process.platform === 'win32') {
```

Comment:
Oh, I guess the difference is smaller because we're not actually waiting for process output?